kuwala/pipelines/google-poi/resources/categories.json

```diff
@@ -602,51 +370,249 @@
+            "cheese_shop",
+            "chicken_shop",
+            "dress_shop",
+            "hat_shop",
+            "salad_shop",
+            "donut_shop",
+            "soft_drinks_shop",
+            "bag_shop",
+            "greeting_card_shop",
+            "cake_shop",
+            "butcher_shop_deli",
+            "trophy_shop",
+            "aquarium_shop",
+            "auto_body_shop",
+            "bird_shop",
             "department_store",
             "dollar_store",
             "variety_store",
+            "musical_instrument_store",
+            "store",
+            "book_store",
+            "wine_store",
+            "appliance_store",
+            "kitchen_furniture_store",
+            "tea_store",
+            "record_store",
+            "cell_phone_store",
+            "electronics_store",
+            "computer_store",
+            "mattress_store",
+            "kitchen_supply_store",
+            "bedding_store",
+            "souvenir_store",
+            "bubble_tea_store",
+            "sportswear_store",
+            "sporting_goods_store",
+            "organic_food_store",
+            "home_improvement_store",
+            "furniture_store",
+            "organic_drug_store",
+            "auto_parts_store",
+            "bedroom_furniture_store",
+            "sofa_store",
+            "office_furniture_store",
+            "organic_store",
+            "christmas_store",
+            "dairy_store",
+            "stereo_store",
+            "luggage_store",
+            "fishing_store",
+            "childrens_clothing_store",
+            "maternity_store",
+            "baby_clothing_store",
+            "childrens_store",
+            "baby_store",
+            "fireplace_store",
+            "mens_clothing_store",
+            "marine_supply_store",
+            "educational_supply_store",
+            "battery_store",
+            "toy_store",
+            "cold_cut_store",
+            "fashion_accessories_store",
+            "natural_foods_store",
+            "gift_basket_store",
+            "jewelry_store",
+            "cosmetics_store",
+            "beauty_supply_store",
+            "craft_store",
+            "health_food_store",
+            "used_clothing_store",
+            "womens_clothing_store",
+            "shoe_store",
+            "general_store",
+            "fruit_and_vegetable_store",
+            "dry_fruit_store",
+            "kiosk",
+            "party_store",
+            "bathroom_supply_store",
+            "pet_supply_store",
+            "reptile_store",
+            "pet_store",
+            "liquor_store",
+            "thrift_store",
+            "tile_store",
+            "outdoor_sports_store",
+            "camping_store",
+            "leather_goods_store",
+            "leather_coats_store",
+            "denim_wear_store",
+            "fish_store",
+            "office_supply_store",
+            "rubber_stamp_store",
+            "stationery_store",
```

Comment:
Please check all of these to see if there are more appropriate categories than "shopping".